CHANGELOG.md

```diff
@@ -62,6 +62,11 @@
 - Added support for `DataFrame.corr`.
 - Added support for `DataFrame.equals` and `Series.equals`.
 - Added support for `DataFrame.reindex` and `Series.reindex`.
+- Added support for `Index.T`.
```

Comment:
I added these here since I think the actual entries for these got wiped out by some other merge?

Reply:
Oh, @sfc-gh-vbudati are those missing in the current release log? you will have to cherry-pick that into release-v1.20.0

Reply:
yeah, they are missing in the current release log